src/debate/debate.js

```diff
@@ -240,7 +275,7 @@ export class Debate {
             return;
         }
 
-        if (this.clients[socket.uuid].answers[questionId] !== undefined) {
+        if (this.getClient(socket.uuid).answers[questionId] !== undefined) {
```

Comment:
We are certain because every client must pass through the initializeClient function. It's the only entry point to be connected to the debate (connection -> initializeClient -> functions).

```
    /**
     * Initialize the client and his attributes
     * @param socket client socket to initialize
     */
    initializeClient(socket) {
        if (this.clients.has(socket.uuid)) {
            logger.debug(`Existing client uuid (${socket.uuid})`)
            this.getClient(socket.uuid).socket = socket;
        } else {
            logger.debug(`New client uuid (${socket.uuid})`)
            // Store the socket and initialize attributes
            this.clients.set(socket.uuid, {
                socket: socket,
                answers: new Map(),
                suggestions: new Set()
            });
            ...
        }
    }
```